makefiles/arch/atmega.inc.mk

```diff
@@ -0,0 +1,39 @@
+# Target architecture for the build. Use avr if you are unsure.
+TARGET_ARCH ?= avr
+
+CFLAGS_CPU   = -mmcu=$(CPU) $(CFLAGS_FPU)
+CFLAGS_LINK  = -ffunction-sections -fdata-sections -fno-builtin -fshort-enums
+CFLAGS_DBG  ?= -ggdb -g3
+CFLAGS_OPT  ?= -Os
+
+CFLAGS    += $(CFLAGS_CPU) $(CFLAGS_LINK) $(CFLAGS_DBG) $(CFLAGS_OPT)
+ASFLAGS   += $(CFLAGS_CPU) $(CFLAGS_DBG)
+LINKFLAGS += $(CFLAGS_CPU) $(CFLAGS_DBG) $(CFLAGS_OPT) -static -lgcc -e reset_handler -Wl,--gc-sections
+OFLAGS    += -j .text -j .data
+
+# Tell the build system that the CPU depends on the atmega common files:
+USEMODULE += atmega_common
+
+# export the peripheral drivers to be linked into the final binary
+USEMODULE += atmega_common_periph
+USEMODULE += periph_common
+
+# Export the peripheral drivers to be linked into the final binary, for now
+# only atmega126rfr2 has periph drivers
+ifeq ($(CPU), atmega256rfr2)
+  USEMODULE += periph
+endif
+
+# the atmel port uses uart_stdio
+USEMODULE += uart_stdio
```

Comment:
typo: this is actually named `stdio_uart`

Reply:
The module was actually renamed in https://github.com/RIOT-OS/RIOT/commit/d55616a7f56ca17440 (so after this PR was opened) so not sure this really is a typo ;-)